src/main/java/duke/Task.java

```diff
@@ -0,0 +1,31 @@
+package duke;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    private TaskType type = TaskType.TASK;
+    protected String time;
+
+    public Task(String description, String time) {
+        this.description = description;
+        this.isDone = false;
+        this.time = time;
+    }
```

Comment:
Maybe you can overload the constructor and make another that accepts an "isDone" as variable.